js/index.js

```diff
@@ -29,11 +21,69 @@ document.addEventListener("DOMContentLoaded", () => {
     "Project Management Systems (Jira, Asana, Wrike)",
     "Email Marketing (Klaviyo, Mailchimp)",
   ];
-  const skillsSection = document.getElementById('skills');
-  skillsList = skillsSection.querySelector('ul');
+  const skillsSection = document.getElementById("skills");
+  skillsList = skillsSection.querySelector("ul");
   for (let i = 0; i < skills.length; i++) {
     const skill = document.createElement("li");
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
   }
+
+
+
+  /////////////////////////////////////////
+  ///////////// MESSAGE FORM /////////////
+  ///////////////////////////////////////
+
+  // HIDE MESSAGES WHEN NONE//
```

Comment:
This is a style preference, but if you're using `//`, feel free to just use it at the start of the line. If you want to pair them, use `/*` and `*/`